packages/onramp-kit/example/client/package.json

```diff
@@ -5,20 +5,23 @@
   "type": "module",
   "scripts": {
     "clean": "rm -rf node_modules/.vite",
-    "start": "yarn clean && vite"
+    "start": "yarn clean && vite",
+    "build": "vite build"
   },
   "dependencies": {
     "@emotion/react": "^11.11.1",
     "@emotion/styled": "^11.11.0",
     "@monerium/sdk": "^2.6.5",
     "@mui/material": "^5.14.17",
+    "@safe-global/auth-kit": "file:../../../auth-kit",
+    "@safe-global/onramp-kit": "file:../../",
+    "@safe-global/protocol-kit": "file:../../../protocol-kit",
     "@safe-global/safe-react-components": "^2.0.5",
-    "@web3auth/modal": "^4.6.2",
-    "@web3auth/openlogin-adapter": "^4.6.0",
     "ethers": "^6.7.1",
     "react": "^18.2.0",
     "react-dom": "^18.2.0",
-    "react-router-dom": "^6.18.0"
+    "react-router-dom": "^6.18.0",
+    "vite-plugin-node-polyfills": "^0.16.0"
   },
   "devDependencies": {
     "@esbuild-plugins/node-globals-polyfill": "^0.2.3",
```

Comment:
In the other example app you removed this devDependencies

Reply:
fixed